packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -455,7 +456,17 @@ const ProductDetail = () => {
         >
             <Helmet>
                 <title>{product?.pageTitle}</title>
-                <meta name="description" content={product?.pageDescription} />
+                {product?.pageMetaTags ? (
```

Comment:
Is there a scenario where `pageMetaTags` is included in the response but is empty, and we would want to fall back to `pageDescription`?

```suggestion
                {product?.pageMetaTags?.length ? (
```

Reply:
I don't think it will ever be empty (the PDP meta data is auto generated based on other properties of the product so I think there'll always be something) but having extra safety is good so I'll add this change.

![Screenshot 2025-02-03 at 3 29 33 PM](https://github.com/user-attachments/assets/72ef3862-e738-40cf-8f2c-ce63f49539b5)